build.gradle

```diff
@@ -46,6 +46,9 @@ dependencies {
     jcenter()
     mavenCentral()
     google()
+    maven {
+      url 'https://oss.sonatype.org/content/repositories/snapshots/'
```

Comment:
Why do we need adding this repo? I would prefer to avoid `SNAPSHOTS` as much as we can.

Reply:
I totally Agree, We actually don't need to add it at all. I added it earlier while following the mixpanel documentation and i thought i removed it before commit.